esrally/metrics.py

```diff
@@ -50,13 +50,13 @@ def __init__(self, client, cluster_version=None):
 
     def put_template(self, name, template):
         tmpl = json.loads(template)
-        return self.guarded(self._client.indices.put_template, name=name, **tmpl)
+        return self.guarded(self._client.indices.put_index_template, name=name, **tmpl)
 
     def template_exists(self, name):
-        return self.guarded(self._client.indices.exists_template, name=name)
+        return self.guarded(self._client.indices.exists_index_template, name=name)
 
     def delete_template(self, name):
-        self.guarded(self._client.indices.delete_template, name=name)
+        self.guarded(self._client.indices.delete_index_template, name=name)
```

Comment:
Nit: I don't think we're using template removal anywhere. Perhaps we could get rid of it?